src/server/models/okta/User.ts

```diff
@@ -78,6 +78,7 @@ export interface UserCreationRequest {
       'email' | 'login' | 'isGuardianUser' | 'registrationPlatform'
     >
   >;
+  groupIds?: Array<string>;
```

Comment:
In `getConfiguration` we use `getOrThrow` so the application won't start without this information. Does that mean we can make this type non optional?

Reply:
This is on the user creation request, technically it's possible to create a user without a groupId, which is why it's optional I think as it matches what the okta documentation says: https://developer.okta.com/docs/reference/api/users/#request-parameters

Reply:
sounds good — i had the wrong context at the time and mistook this for the general configuration type

Reply:
we won't have permissions to execute the api call without setting this param, so we could also make it mandatory here and make it a build time error instead of runtime error ?

Reply:
Yeah, it might be a good idea to make it mandatory for that reason 👍🏾

Reply:
I made it mandatory now